epsagon/tracer.go

```diff
@@ -77,10 +77,14 @@ func (tracer *epsagonTracer) sendTraces() {
 
 	resp, err := client.Post(tracer.Config.CollectorURL, "application/json", tracesReader)
 	if err != nil {
-		var respBody []byte
-		resp.Body.Read(respBody)
-		resp.Body.Close()
-		log.Printf("Error while sending traces \n%v\n%+v\n", err, respBody)
+		if resp != nil {
```

Comment:
This is a good change but not part of migrating the uuid library, can you split it to another commit at least?

Reply:
agree, let me move it into another pr. I had to do it because the tests were failing for me.